4CE_Phase1.1_Files_oracle.sql

```diff
@@ -300,18 +326,20 @@ commit;
 --   and then find all the concepts corresponding to child paths.
 -- WARNING: This query might take several minutes to run. If it is taking more
 --   than an hour, then stop the query and contact us about alternative approaches.
-/*
+--drop table COVID_MED_PATHS;
+-- select distinct med_class from COVID_MED_PATHS order by med_class;
 create table COVID_MED_PATHS AS
 select concept_path, concept_cd
-	from concept_dimension
-	where concept_path like '\ACT\Medications\%'
-		and concept_cd in (select concept_cd from observation_fact); 
-alter table COVID_MED_PATHS add constraint COVID_MEDPATHS_PK primary key (concept_path);
+	from nightherondata.concept_dimension
+	where concept_path like '\ACT\Medications\MedicationsByVaClass\V2_09302018\%' -- '\ACT\Medications\MedicationsByAlpha\V2_12112018\RxNormUMLSRxNav\%' --
+		and concept_cd in (select concept_cd from nightherondata.observation_fact); 
+-- TODO: enable concept_path primary key
+--alter table COVID_MED_PATHS add constraint COVID_MEDPATHS_PK primary key (concept_path);
 alter table COVID_MED_PATHS add med_class varchar(50);
 insert into COVID_MED_PATHS
 	select distinct 'Expand', d.concept_cd, m.med_class
 	from COVID_MED_MAP m
-		inner join concept_dimension c
+		inner join nightherondata.concept_dimension c
```

Comment:
hard-coding schema names seems awkward. I wonder if I managed to avoid that before. I guess it doesn't hurt that much. But I wonder if aliases would be better.